seed.sql

```diff
@@ -1,23 +1,33 @@
 \c country_game;
 
-INSERT INTO faction (name) VALUES 
- ('USA')
-,('Russa')
-,('China')
-,('Canada')
-,('Mexico')
-,('North Korea')
-,('Great Britain')
+INSERT INTO faction (id, name) VALUES 
+ ('10000000-a234-4729-bd2a-68379df315fb', 'USA')
+,('20000000-a234-4729-bd2a-68379df315fb', 'Russa')
+,('30000000-a234-4729-bd2a-68379df315fb', 'China')
+,('40000000-a234-4729-bd2a-68379df315fb', 'Canada')
+,('50000000-a234-4729-bd2a-68379df315fb', 'Mexico')
+,('60000000-a234-4729-bd2a-68379df315fb', 'North Korea')
+,('70000000-a234-4729-bd2a-68379df315fb', 'Great Britain')
 ;
 
-INSERT INTO regions (click_count, is_parent) VALUES
- (1000000, true)
-,(1488, false)
-,(99999, false)
-,(123456, false)
-,(8888888, false)
-,(-2, false)
-,(3, true)
+INSERT INTO regions (id, click_count, name) VALUES
+ ('10000000-a234-4729-bd2a-68379df315fb', 1000000, 'Oregon')        -- USA
+,('20000000-a234-4729-bd2a-68379df315fb', 1488, 'California')       -- USA
+,('30000000-a234-4729-bd2a-68379df315fb', 99999, 'Moscow')          -- RUSSIA
+,('40000000-a234-4729-bd2a-68379df315fb', 123456, 'Coahuila')       -- MEXICO
+,('50000000-a234-4729-bd2a-68379df315fb', 8888888, 'Devonshire')    -- GB
+,('60000000-a234-4729-bd2a-68379df315fb', -2, 'Pyongyang')          -- NC
+,('70000000-a234-4729-bd2a-68379df315fb', 3, 'Manchuria')           -- CHINA
+;
+
+INSERT INTO region_ownership (region_id, faction_id) VALUES
```

Comment:
There is no need to make a `region_ownership` table. It would be much more logical to make a `faction` column in the region table that is the uid of its faction.